finance.md

```diff
@@ -2,70 +2,113 @@
-Our TransferWise account has local accounts in Canada, United States, and Europe, so we can bill clients as a local financial institution in their local currency.
+Our TransferWise account has local accounts in Canada, United States, Europe, and the United Kingdom, so we can bill clients as a local financial institution in their local currency.
 Our TransferWise account does not require multiple signatures, and the Finance WG is responsible for transferring money into our main Desjardins account when accumulated funds exceed a CAD 10,000 threshold, or earlier as deemed appropriate.
 
 ## Invoicing Clients
 
 Hypha charges clients on a sliding scale, and projects may be billed hourly, weekly, monthly, or per project, depending on the specifics of each project.
-In order to create invoices and collect payments, there are three general steps that need to be performed in order.
+The recommended practice is to send an invoice every month, or a quote if the amount for the month is small and at the discretion of the project.
+
+In cases where an invoice is not collected in time for payroll, or failed to be collected altogether, this is treated as a cost to the Co-operative and we strongly advocate that this not affect the timely payroll to the Member.
+
+In order to create invoices and to collect payments, there are three general steps that need to be performed in order.
 
 ### Generating Timesheets
 
-For clients with billing models that require tracked work hours, **Clockify** is used to generate timesheets for determining the invoice amount each period, and often for calculating member wages.
+For clients with billing models that require tracked work hours, **Clockify** is used to generate timesheets for determining the invoice amount each period, and often for calculating Member wages.
 These timesheets are archived in our [shared drive](https://link.hypha.coop/drive) under the `Timesheets` directory and may be attached to invoices at client request.
 
 Refer to the guides for [Timesheets](/guides.md#timesheets) to generate and archive your project timesheets.
 
 ### Generating Invoices
 
 Refer to the guides for [Invoices](/guides.md#invoices) and generate an invoice in **Quickbooks Online** based on the project pricing structure (e.g. timesheet hours and project rate) and payment information.
-The general guidance is that the client is responsible for paying applicable GST and transaction costs charged by their financial institution, and Hypha absorbs the cost of receiving the money (e.g. cost of receiving a wire).
-Invoices, with attached timesheets if requested, are sent to clients and are due 14 days after the invoice date, after which a late fee may be charged for each additional week.
+
+In general, the client is responsible for paying applicable HST and transaction costs charged by their financial institution, and Hypha pays the cost of receiving the payment (e.g. cost of receiving a wire).
```

Comment:
```suggestion
In general, the client is responsible for paying transaction costs charged by their financial institution, and Hypha pays the cost of receiving the payment (e.g. cost of receiving a wire).
```

seemed extraneous?

Reply:
I think it's better to be explicit? Seems obvious to us now, but wasn't in the beginning :)